etna/core/wagtail_hooks.py

```diff
@@ -29,3 +32,14 @@ def global_admin_css():
     if settings.FEATURE_PLATFORM_ENVIRONMENT_TYPE != "production":
         return "<style> @media (prefers-color-scheme: light) { :root {--w-color-primary: #00623B; --w-color-primary-200: #003c1e;} } @media (prefers-color-scheme: dark) { :root {--w-color-surface-menus: #002510; --w-color-surface-menu-item-active: #001810;} }</style>"
     return ""
+
+
+class TaxonomiesAdminGroup(ModelViewSetGroup):
+    menu_label = "Taxonomies"
+    items = (series_viewset,)
+    menu_icon = "tag"
+
+
+@hooks.register("register_admin_viewset")
+def register_series_viewset():
+    return TaxonomiesAdminGroup()
```

Comment:
Do we need a specific menu tab for this? I think ultimately it won't be too big of a piece to have in the snippets menu area for example, but I'm happy with it as it is.

Reply:
I've just turned the whole block into a snippet anyway. In the future, we might want to consider using taxonomy models, but it's probably an overkill for now.

Reply:
Interesting, thank you, I didn't know that was a thing! 😄